web/packages/dmt-core/src/context/FileSystemTreeContext.tsx

```diff
@@ -19,6 +19,7 @@ export const FSTreeContext = createContext<{
 
 export const FSTreeProvider = (props: { children: ReactNode }) => {
   const { children } = props
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore